sdk/core/core-tracing/package.json

```diff
@@ -37,9 +37,9 @@
   },
   "files": [
     "dist/",
-    "dist-esm/lib/",
+    "dist-esm/src/",
+    "src/",
```

Comment:
this duplicates the below line, but also why do we include the source files in the package to begin with?

Reply:
I don't know - it shouldn't be there - we don't have to ship src with our package

Reply:
agreed, I think you can take it out

Reply:
removed src, let me know if that's ok?